src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,136 @@
+        assertThatThrownBy(() -> menuService.create(expected))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("메뉴의 가격이 0원보다 작으면 등록할 수 없다.")
+    @Test
+    void cantCrateOneMenuWhenPriceUnderZero() {
+        List<MenuProduct> menuProducts = Arrays.asList(
+                new MenuProduct(1L, 1L, 10L),
+                new MenuProduct(1L, 2L, 10L)
+        );
+        Menu expected = new Menu("메뉴1", new BigDecimal(-1), 1L, menuProducts);
+        assertThatThrownBy(() -> menuService.create(expected))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("메뉴의 가격이 메뉴에 속한 상품금액의 합보다 크면 exception")
+    @Test
+    void menuPriceTest() {
+        MenuGroup group = menuGroupDao.save(new MenuGroup("그룹1"));
+        Product product1 = productDao.save(new Product("상품1", new BigDecimal(1000)));
+        Product product2 = productDao.save(new Product("상품2", new BigDecimal(2000)));
+        List<MenuProduct> menuProducts = Arrays.asList(
+                new MenuProduct(1L, product1.getId(), 2L),
+                new MenuProduct(1L, product2.getId(), 1L)
+        );
+        menuProductDao.save(menuProducts.get(0));
+        menuProductDao.save(menuProducts.get(1));
+
+        Menu expected = new Menu("메뉴1", new BigDecimal(5000), group.getId(), menuProducts);
+        assertThatThrownBy(() -> menuService.create(expected))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("메뉴는 특정 메뉴 그룹에 속해야 한다.")
+    @Test
+    void menuIncludedGroupTest() {
+        Product product = productDao.save(new Product("상품1", new BigDecimal(1000)));
+        List<MenuProduct> menuProducts = Arrays.asList(new MenuProduct(1L, product.getId(), 2L));
+        menuProductDao.save(menuProducts.get(0));
+
+        Menu expected = new Menu("메뉴1", new BigDecimal(5000), 1L, menuProducts);
+        assertThatThrownBy(() -> menuService.create(expected))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("메뉴의 목록을 조회할 수 있다.")
+    @Test
+    void listMenus() {
+        MenuGroup group = menuGroupDao.save(new MenuGroup("그룹1"));
+        Product product1 = productDao.save(new Product("상품1", new BigDecimal(1000)));
+        Product product2 = productDao.save(new Product("상품2", new BigDecimal(2000)));
+        Menu menu = new Menu("메뉴1", new BigDecimal(10000), group.getId(), null);
+        Menu save = menuDao.save(menu);
+        MenuProduct menuProduct1 = menuProductDao.save(new MenuProduct(save.getId(), product1.getId(), 1));
+        MenuProduct menuProduct2 = menuProductDao.save(new MenuProduct(save.getId(), product2.getId(), 2));
+        List<MenuProduct> menuProducts = Arrays.asList(menuProduct1, menuProduct2);
+        save.setMenuProducts(menuProducts);
+
+        List<Menu> results = menuService.list();
+        assertThat(results.size() > 1).isTrue();
```

Comment:
assertj 에서 제공하는 기능을 활용하는 것도 좋겠네요

```suggestion
        assertThat(results.size()).isGreaterThan(1);
```